src/cross_section.F90

```diff
@@ -248,6 +267,12 @@ subroutine calculate_nuclide_xs(i_nuclide, i_sab, E, i_mat, i_nuc_mat, u)
       micro_xs(i_nuclide) % kappa_fission = &
            nuc % reactions(nuc % index_fission(1)) % Q_value * &
            micro_xs(i_nuclide) % fission
+
+      ! Calculate microscopic nuclide delayed nu-fission cross section
+      micro_xs(i_nuclide) % delayed_nu_fission = (ONE - f) * &
+           nuc % delayed_nu_fission(i_grid) + f * &
+           nuc % delayed_nu_fission(i_grid+1)
```

Comment:
I used gprof to profile the code with and without the computation of the microscopic delayed-nu-fission cross section in the `calculate_nuclide_xs` subroutine. I ran the `examples/xml/lattice/simple` case with 1e6 particles/batch three times with both versions of the code. The `calculate_nuclide_xs` subroutine took up ~4-5% of the total run time for both code versions. In the cases that include the calculation of the microscopic delayed-nu-fission xs, the run times for the `calculate_nuclide_xs` subroutine were 10.03, 8.88, and 9.77 seconds for an average of 9.5 seconds. In the cases that did not include the calculation of the microscopic delayed-nu-fission xs, the run times for the `calculate_nuclide_xs` subroutine were 9.84, 9.12, and 9.00 seconds for an average of 9.3 seconds. All cases were run on an AWS m4.10xlarge instance with one MPI thread. This is clearly a small sample size, but nevertheless it seems to indicate that the calculation of the microscopic delayed-nu-fission xs does not have a significant impact on run time.

Is there a specific input you would recommend I run in profile mode to get a more representative indication for the impact of adding the calculation of the microscopic delayed-nu-fission xs?